cd-menu/cd-menu.js

```diff
@@ -134,7 +135,24 @@ window.cdMenu = function (options) {
         loginRegister.style.display = 'block';
       });
     }
-
+    function canSeeELearningModule (userDojos) {
```

Comment:
Only other option I see is to have two separate functions isMentor, and isChampion, but then we're adding unnecessary loops. I'm ok with this until we have another use case that makes sense to generalise for.